### PR TITLE
Removes donut chart and adds stacked area chart

### LINKED ISSUE #812: Remove donut chart and open proposal table in favour of stacked area chart of signed proposals filtered over time

### PR DESCRIPTION
**Pull request by devpavan04**
## Summary of changes
- Removes donut chart and open proposal table from proposals page
- Adds stacked area chart for total proposal count over time

### Reference issue to close (if applicable)
- Closes #812

### CODE REVIEW

**Comment by dutterbutter** [comment]
@devpavan04 can we make 3M the default filter on the chart so it takes up more of the chart space. In addition, can you add hover states on the chart to display the actual count of the proposals? For example, if I hover over Refresh proposal it should display 1. 

Great work!

File: libs/webb-ui-components/src/components/Chip/types.ts [hunk 1]
```diff
@@ -25,12 +25,15 @@ export interface ChipProps extends WebbComponentBase {
    * If `true`, the chip will display as disabled state
    */
   isDisabled?: boolean;
+  isSelected?: boolean;
+  onClick?: React.EventHandler<React.MouseEvent | React.KeyboardEvent>;
```
Comment (AtelyPham) [inline libs/webb-ui-components/src/components/Chip/types.ts:29]: The `onClick` is already inside the `WebbComponentBase` interface. We don't need to redefine it here.

**Comment by devpavan04** [comment]
### Changes overview

https://user-images.githubusercontent.com/53374218/207389447-2b62d9df-2e48-4864-9ed3-9329d2225415.mov

**Comment by AtelyPham** [comment]
Can we add the hover state for legends on the right? When you hover one legend on the right, it will pop-up the count of that proposal. Can you confirm that? @dutterbutter 

Also, update the chart labels color to `text-mono-100`. @devpavan04 

CleanShot 2022-12-13 at 23 45 14@2x

File: apps/stats-dapp/src/gql/proposals.graphql [hunk 2]
```diff
@@ -178,3 +205,117 @@ query ensureProposals($ids: [String!]!) {
     }
   }
 }
+query ProposalsOvertimeTotalCount($start: Int!, $end: Int!) {
```
Comment (AhmedKorim) [inline apps/stats-dapp/src/gql/proposals.graphql:208]: Please add comments to describe the parameters

File: apps/stats-dapp/src/provider/hooks/useProposals.ts [hunk 3]
```diff
@@ -605,3 +607,106 @@ export function useVotes(votesReqQuery: VotesQuery): VotesPage {
...
+
+  const client = useApolloClient();
+
+  const {
+    blockTime,
+    metaData: { lastProcessBlock },
+  } = useStatsContext();
+
+  const blockRanges: BlockRanges = useMemo(() => {
+    let _blockRanges: BlockRanges = [];
```
Comment (AhmedKorim) [inline apps/stats-dapp/src/provider/hooks/useProposals.ts:647]: The `_` before a variable name normally means that the variables isn't in use please remove it

File: apps/stats-dapp/src/provider/hooks/useProposals.ts [hunk 4]
```diff
@@ -605,3 +607,106 @@ export function useVotes(votesReqQuery: VotesQuery): VotesPage {
...
+          setProposalsOvertimeCount({
+            isLoading: false,
+            isFailed: false,
+            val: arr,
+          });
+        }
+      })
+      .catch((e) => {
+        setProposalsOvertimeCount({
+          isLoading: true,
```
Comment (AhmedKorim) [inline apps/stats-dapp/src/provider/hooks/useProposals.ts:703]: `isLoading` should be `false` here

File: apps/stats-dapp/src/provider/hooks/useProposals.ts [hunk 5]
```diff
@@ -605,3 +607,106 @@ export function useVotes(votesReqQuery: VotesQuery): VotesPage {
...
+
+  useEffect(() => {
+    Promise.all(
+      blockRanges.map((parameter) => {
+        return client.query({
+          query: ProposalsOvertimeTotalCountDocument,
+          variables: parameter,
+        });
+      })
+    )
```
Comment (AhmedKorim) [inline apps/stats-dapp/src/provider/hooks/useProposals.ts:677]: Before doing the mapping from `parameters` to `query` you should set the state to be `isLoading= true`

File: apps/stats-dapp/src/utils/getBlockRanges.ts [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    timeRanges.push({
+      start: startBlockNumber,
+      end: endBlockNumber,
+    });
+  }
+
+  return timeRanges;
+};
```
Comment (AhmedKorim) [inline apps/stats-dapp/src/utils/getBlockRanges.ts:85]: Please add more comments documenting this function

**Review by AhmedKorim — CHANGES_REQUESTED** [review]
Great job, Just small changes and documentation missing

**Comment by devpavan04** [comment]
### Changes overview

https://user-images.githubusercontent.com/53374218/207727553-bca943d0-14a5-420e-8fc7-708a4e22456f.mov

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**